### PR TITLE
Added post about website construction

### PR DESCRIPTION
**Pull request by GFLEMING133**
Ran an update on bundler (sorry for the 20 changed files)
& added a post about website construction

### CODE REVIEW

**Comment by hunterparks** [comment]
I also changed some wording in the post - once the `.sass-cache` stuff is cleaned up, this is ready for approval!

File: _posts/2018-01-18-welcome-to-the-new-sse-website.md [hunk 1]
```diff
@@ -9,16 +9,16 @@ hero: https://source.unsplash.com/collection/145103/
 overlay: orange
 published: true
 ---
-I'm glad you're here! We have some amazing stuff going on!
+I'm glad you're here! We have some amazing stuff going on!git log
```
Comment (hunterparks) [inline _posts/2018-01-18-welcome-to-the-new-sse-website.md]: Was the `git log` on this line intentional?

**Comment by hunterparks** [comment]
Hey @GFLEMING133 - did you intend to delete all of the files in the `_sass/` folder? I thought those were all the styles, but I could be mistaken.

**Comment by GFLEMING133** [comment]
Ah, no, definitely did not mean to do that--the _sass folder DOES hold all the styles & is therefore kind of important. Is it possible to cherry-pick on github?

________________________________

**Comment by hunterparks** [comment]
I do not know if you can pick which commits to merge, but I think you could probably just change back to master, grab the folder, change back to your branch and add the folder. A bit of a cave-person approach, but it should work.

**Comment by sorianog** [comment]
@GFLEMING133 We were still waiting on this, correct?

**Pull request closed without merging**